packages/manager/modules/vps/src/veeam/vps-veeam.html

```diff
@@ -7,11 +7,58 @@
     </div>
 
     <div data-ng-if="!$ctrl.veeam.loading">
+
+        <!--Backup Schedule section-->
+        <div class="mb-3" data-ng-if="$ctrl.isAutomateBackupAvailable">
+
+            <!--Schedule automate backup-->
+            <div
+                class="row align-items-center"
+                data-ng-if="$ctrl.isValidToSchedule()"
+            >
+                <!--Schedule text description-->
+                <div class="col-auto">
+                    <p class="mb-0">
```

Comment:
Does `<p>` element needed?

Reply:
@antleblanc, i think yes but i am not sure.
I think for the `a11y`, it must be like this to allow the screen reader to consider the whole p as common text.
`p = span(description) + span(time)`.

what do you think ?

Reply:
I think `p` allows the screen reader to skip the whole content if the user wants to, i'm not sure about spans though.
Also I think the span should use a `d-block` class instead of using `br` tag for semantic reasons

Reply:
Thanks @marie-j , i use the `d-block` class and i removed the `br` tag.
I searched to know what is the a11y good practice for this case but i don't really found a response. 
So i let it as know.